packages/@coorpacademy-components/src/molecule/learning-priority-card/types.ts

```diff
@@ -15,5 +16,6 @@ export const propTypes = {
     label: PropTypes.oneOf(['Playlist', 'Certification', 'Skill'])
   }),
   progression: PropTypes.number,
-  onClick: PropTypes.func
+  onClick: PropTypes.func,
+  type: PropTypes.oneOf(['learningPriority'])
```

Comment:
Maybe we can define a `type` prop in `CardsList` component instead of in the card itself? Cause the card itself doesn't have many type, but the CardsList does (normal card and learningPriority card)

Reply:
yes, it was what I've done initially but in order to not interfering with the existing propsTypes "type" (which it's not mention in CardLists propTypes as mention above) and to not add an new props only for this case,
I used a type in the card itself (to maintain the same logic)

I don't wanna change/break existing components behavior

Reply:
maybe we can try this since we now add `type` props to `Cardlist`? (just a suggestion)